pages/lua_manual/getting_started.md

```diff
@@ -19,12 +19,16 @@ Tested with Lua 5.3 which is the recommended version, but we aim for compatibili
 
 Perform the following steps to integrate the Lua plugin with RmlUi.
 
-1. Download and build the Lua 5.x library, or install it with your package manager.
+1. Install or build a Lua library. You may use the official lua.org implementation or LuaJIT. With the official implementation, you should prefer to build it as C++ rather than C
+   so that the stack can be unwound properly in case of an error. When built as C, the Lua interpreter calls longjmp when an error occurs, which causes the destructors
+   for local variables in any currently executing C++ extension functions to be skipped. Package managers often provide liblua in C mode only, so you will likely
+   have to build it as C++ yourself.
```

Comment:
First of all, thanks a lot for contributing! It's a nice improvement. I'd like to make some tweaks:

```suggestion
1. Install or build the Lua library, you may use the official [Lua.org](https://www.lua.org) implementation or [LuaJIT](http://luajit.org/luajit.html).
    - With the official implementation, we recommend to build the library as C++ rather than C so that the stack can be unwound properly in case of an error. When built as C, the Lua interpreter calls longjmp when an error occurs, which causes the destructors for local variables in any currently executing C++ extension functions to be skipped. Package managers typically provide Lua compiled in C only, so for this functionality you will have to build it as C++ yourself.
```
Some formatting and rewording, slightly less strongly worded recommendation. Does it look okay to you?

Reply:
Looks good.